<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: #90cdf4;
}</string>
   </property>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>370</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #3182CE;
border: none;
border-radius: 20px;
font-size: 12px;
color: white;
font-weight: 600;
}

QPushButton:hover {
background-color: #90CDF4;

}

QPushButton:pressed {
background-color: #90CDF4;

}
</string>
    </property>
    <property name="text">
     <string>Stop/Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #3182CE;
border: none;
border-radius: 20px;
font-size: 12px;
color: white;
font-weight: 600;
}

QPushButton:hover {
background-color: #90CDF4;

}

QPushButton:pressed {
background-color: #90CDF4;

}
</string>
    </property>
    <property name="text">
     <string>Start Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="screenLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>624</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #3182CE;
border: none;
border-radius: 20px;
font-size: 12px;
color: white;
font-weight: 600;
}

QPushButton:hover {
background-color: #90CDF4;

}

QPushButton:pressed {
background-color: #90CDF4;

}
</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: white;
	font-size: 16px;
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>Images</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listImages">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeImage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #3182CE;
border: none;
border-radius: 20px;
font-size: 12px;
color: white;
font-weight: 600;
}

QPushButton:hover {
background-color: #90CDF4;

}

QPushButton:pressed {
background-color: #90CDF4;

}
</string>
    </property>
    <property name="text">
     <string>Remove Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slideShow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #3182CE;
border: none;
border-radius: 20px;
font-size: 12px;
color: white;
font-weight: 600;
}

QPushButton:hover {
background-color: #90CDF4;

}

QPushButton:pressed {
background-color: #90CDF4;

}
</string>
    </property>
    <property name="text">
     <string>Slide Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="postion">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="slide" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
background-color: #90cdf4;
}</string>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="end">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>340</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
